### ISSUE TITLE
Positioning

### ISSUE THREAD

**Issue by Northernn** [tomek-o/tSIP#39]
I found an odd issue. 
I was working on a laptop on multiple screens, so moved programs onto the other screens, including the phone software.
Next day I am in a new location and no access to the multiple screens, but I open everything, and all porgrams reset themselves onto the one screen, except for tsip, I cannot get it back onto the main screen! it is stuck on the second screen which I don't have access to!

I think it needs to have a reset position somewhere, either automatic when it detects one screen only or maybe in the right click menu on the icon, that will just move it to the center of the main display.

I am guessing but not looked yet that I can sort it out somewhere in one of the json or ini files, so will go look into that now.

**Comment by Northernn** [comment]
I reset it by replacing the frmMain section in the json file which fixed the issue! I think it was the PositionX and PositionY that needed a reset, maybe a quick check on load for if it is outside the visiable area and no extra screens detected? not sure if this would work.
Thanks

**Comment by tomek-o** [comment]
I have not reproduced this at the moment on Windows 7. Clicking on taskbar button brought window to the first monitor. Not sure if this is obvious, but I got accustomed to this as it works same way for Firefox. I don't have second monitor connected on Windows 10 right now, doesn't it work the same way?

**Comment by Northernn** [comment]
It was on windows 11, on 10 I can right click and move, then use the arrow keys to move the window back on screen, but that option is gone in windows 11. it went from 2x 27" screens to a laptop 15" screen.  All the other programs re-positioned on the main screen and all the icons did too, but tsip doesn't, just makes it a little difficult.

**Comment by tomek-o** [comment]
On Windows 7 I saw that the event that brings windows to active monitor is restoring from minimized state and it would be very strange if tSIP would behave in the other way than all other applications. Perhaps you have tested it by clicking on the tray icon, not the taskbar button.
Anyway, in the latest pre-release from feature-multiple-calls I've added checking for missing monitor to application OnRestore event (in my tests it was unnecessary) and to restoring application from tray.
